dbms/src/main/java/org/polypheny/db/processing/AbstractQueryProcessor.java

```diff
@@ -1400,7 +1399,7 @@ private void monitorResult( ProposedRoutingPlan selectedPlan ) {
      *
      * @param eventData monitoring data to be updated
      */
-    private void finalizeAccessedPartitions( StatementEvent eventData ) {
+    public void finalizeAccessedPartitions( StatementEvent eventData ) {
```

Comment:
Why changing the visibility to public? This method is only called from within this class.

Reply:
Changed back to private.